modules/java/partials/release-notes/couchbase-mobile-java-release-note.3.0.5.adoc

```diff
@@ -1,8 +1,3 @@
-
-
-
-
-
 [#maint-3-0-5]
```

Comment:
Might be even better to delete this file, as it's 3.0 and hidden in the 3.1 release branch